kitsune/community/jinja2/community/email/first_answer.ltxt

```diff
@@ -34,7 +34,7 @@ We'd love to get to know you! If you feel comfortable, say hi and introduce
 yourself in this thread.
 {% endtrans %}
 
-https://{{ host }}/forums/contributors
+https://mzl.la/4ll91QX
```

Comment:
This one also seems to contain the wrong target URL? It doesn't go to `forums/contributors`.